gpu-service/Dockerfile

```
@@ -0,0 +1,75 @@
+    libgtk-3-dev \
+    libboost-all-dev \
+    libblas-dev \
+    libatlas-base-dev \
+    gfortran \
+    git \
+    ffmpeg \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install newer CMake
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.25.0/cmake-3.25.0-linux-x86_64.tar.gz \
```

Hadolint (2.12.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)